docs/supported_platforms_and_configurations.md

```diff
@@ -51,9 +51,11 @@ For more information see [conan-io/conan-docker-tools](https://github.com/conan-
   - Shared, Static (option `"shared": [True, False]` in the recipe when available)
   - Header Only (option `"header_only": [True, False]` if available)
 
+> :warning: The profile with the option `shared=True` and runtime `MT/MTd` is not built.
+
 ## Linux
 
-- Python: 3.7.5, 3.7.13, 3.8.2 (same version expected after all use [new docker images](https://github.com/conan-io/conan-docker-tools/tree/master/modern))
+- Python: 3.7.13 (same version expected after all use [new docker images](https://github.com/conan-io/conan-docker-tools/tree/master/modern))
```

Comment:
Expected after https://github.com/conan-io/conan-docker-tools/pull/393, probably there is no need for the extra note now (I'll remove it in the template)